cms/templates/maintenance/_announcement_index.html

```diff
@@ -0,0 +1,49 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="studio-view maintenance-form">
+    <div class="form-create">
+        <div class="announcement-container">
+            % for announcement in announcement_list:
+                <div class="announcement-item">
+                    <div class="announcement-content">
+                        ${announcement.content | n} ## xss-lint: disable=mako-invalid-html-filter
```

Comment:
[Ditto](https://github.com/edx/edx-platform/pull/16496/files#r251327119) for these `xss-lint` comments too.